### PR TITLE
Settings page & functionality

### PR DESCRIPTION
**Pull request by kainanh**

### CODE REVIEW

File: src/Networking.js [hunk 1]
```diff
@@ -51,4 +55,40 @@ export default class Networking {
     const data = await response.json();
     return data;
   }
+
+  async updateInformation(name, weight, height, age, gender) {
+    const userInformation = { name, weight, height, age, gender };
```
Comment (dan-michell) [inline src/Networking.js:60]: Possible change to more descriptive name?

File: src/Networking.js [hunk 2]
```diff
@@ -51,4 +55,40 @@ export default class Networking {
...
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify(userInformation),
+    });
+    const data = await response.json();
+    return data;
+  }
+
+  async updateGoals(calories, protein, carbs, fats, sugar, salt, fibre) {
+    const userInformation = {
```
Comment (dan-michell) [inline src/Networking.js:74]: Possible change to more descriptive name?

File: src/Networking.js [hunk 3]
```diff
@@ -51,4 +55,40 @@ export default class Networking {
     const data = await response.json();
     return data;
   }
+
+  async updateInformation(name, weight, height, age, gender) {
+    const userInformation = { name, weight, height, age, gender };
+    const response = await fetch(`${process.env.REACT_APP_API_URL}/user-info`, {
+      method: "POST",
```
Comment (dan-michell) [inline src/Networking.js:62]: Method should be patch?

File: src/Networking.js [hunk 4]
```diff
@@ -51,4 +55,40 @@ export default class Networking {
...
+      method: "PATCH",
+      credentials: "include",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify(userInformation),
+    });
+    const data = await response.json();
+    return data;
+  }
```
Comment (dan-michell) [inline src/Networking.js:93]: I think we also need to add two more functions to add an entry to the goals and user-info table. You can basically just copy and paste the two functions you have made and change the methods to post. With the goals post function you don't need to send any information in the body as we are just adding the default values to the table initially.

**Comment by Ibahmed1** [comment]
everything looks good. Could we change the file name Settings form to infoForm to avoid confusion?

**Pull request closed without merging**